src/usr/local/www/status_logs_filter.php

```diff
@@ -200,14 +391,18 @@ function build_if_list() {
 $tab_array[] = array(gettext("Summary View"), false, "/status_logs_filter_summary.php");
 display_top_tabs($tab_array, false, 'nav nav-tabs');
 
-$Include_Act = explode(",", str_replace(" ", ",", $filterfieldsarray['act']));
-if ($filterfieldsarray['interface'] == "All")
-	$interface = "";
+define(SEC_OPEN, 0x00);
+define(SEC_CLOSED, 0x04);
 
```

Comment:
Since these are defined in classes/Form/Section.class.php I don't think they are required here.